README.md

```diff
@@ -162,7 +162,8 @@ Here's the enrollment process from the owner's perspective:
 
 In order for this process to work, you must register a domain name that
 identifies your application. The Tesla app will display this domain name to the
-user when it asks if they wish to approve your request.
+user when it asks if they wish to approve your request, and the vehicle will
+use the domain name to identify the key in the Locks screen.
```

Comment:
This wording doesn't make it quite clear that the domain name is actually displayed. Maybe something like `vehicle will show the domain name with the key in the Locks screen`?